apps/ledger-live-desktop/tests/enum/AppInfos.ts

```diff
@@ -0,0 +1,126 @@
+import { DeviceLabels } from "./DeviceLabels";
+
+export class AppInfos {
+  constructor(
+    public readonly name: string,
+    public readonly sendPattern: DeviceLabels[],
+    public readonly receivePattern: DeviceLabels[],
+  ) {}
+  static readonly BITCOIN = new AppInfos(
+    "Bitcoin",
```

Comment:
why don't you use Currency.BITCOIN.name here?

Reply:
Because in some cases, the Currency.name is not the same (e.g., XRP/Ripple), and for apps that don't use Currency, like Ledger Sync, I thought it was better to separate the two.